src/framework/application.js

```diff
@@ -1748,11 +1739,11 @@ class Application extends EventHandler {
      * });
      */
     renderLine(start, end, color) {
-        var endColor = color;
-        var options;
+        let endColor = color;
```

Comment:
these render functions are all now moved to deprecated in my other PR.

Reply:
So what needs to be done here?

Reply:
nothing for now I think .. we'll resolve to conflict when we get to it.

Reply:
ok